### PR TITLE
Remove front-end from jest projects

### LINKED ISSUE #3643: Jest crashes with Nextjs v12.1.6
**What happened**:
Since our [update to Nextjs 12.1.6](https://github.com/Seneca-CDOT/telescope/commit/7ea9e5a77aa8a744f63b9d74ecb86f99f00e7bd8), the script `pnpm jest` has been crashing, giving an error log below.

```
Error: > Couldn't find a `pages` directory. Please create one under the project root
    at Object.findPagesDir (C:\OSD\telescope\node_modules\.pnpm\next@12.1.6_tmacpkxga3bcx5k2p7mjqnxqbe\node_modules\next\dist\lib\find-pages-dir.js:39:15)
    at C:\OSD\telescope\node_modules\.pnpm\next@12.1.6_tmacpkxga3bcx5k2p7mjqnxqbe\node_modules\next\dist\build\jest\jest.js:26:47
    at readConfigFileAndSetRootDir (C:\OSD\telescope\node_modules\.pnpm\jest-config@27.5.1\node_modules\jest-config\build\readConfigFileAndSetRootDir.js:152:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)    at async readConfig (C:\OSD\telescope\node_modules\.pnpm\jest-config@27.5.1\node_modules\jest-config\build\index.js:233:18)
    at async Promise.all (index 0)
    at async readConfigs (C:\OSD\telescope\node_modules\.pnpm\jest-config@27.5.1\node_modules\jest-config\build\index.js:441:27)
    at async runCLI (C:\OSD\telescope\node_modules\.pnpm\@jest+core@27.5.1\node_modules\@jest\core\build\cli\index.js:132:59)
    at async Object.run (C:\OSD\telescope\node_modules\.pnpm\jest-cli@27.5.1\node_modules\jest-cli\build\cli\index.js:155:37)
 ELIFECYCLE  Command failed with exit code 1.
```

**What should have happened**:
Jest tests should run

**How to reproduce it (as precise as possible)**:
Run `pnpm jest`

**Anything else we need to know?**:
The error happens [on this line](https://github.com/vercel/next.js/blob/8258c872c4b3d3c336c4811ddb570d72e78455bb/packages/next/lib/find-pages-dir.ts#L38) in Next.

What happened in Next is they [added a new `pagesDir` in their Jest transformer](https://github.com/vercel/next.js/blob/8258c872c4b3d3c336c4811ddb570d72e78455bb/packages/next/build/jest/jest.ts#L59). 

In our monorepo, Jest considers the Telescope root as the root directory, instead of the web app root, and since it can't find `pages` in `<root>` or `<root>/src`, an error is thrown and the crash occurs.

Our choices may be
1. Somehow tweak the front-end Jest config so it recognizes the web app root as root.

2. Temporarily comment out [this line](https://github.com/Seneca-CDOT/telescope/blob/7e852e9388ae991a27871e525805f0d4600b9c11/jest.config.js#L7) in the Telescope root Jest config projects list. 
This will let all other unit tests to run in Jest. 
Currently we have [a single unit test](https://github.com/Seneca-CDOT/telescope/blob/master/src/web/app/src/components/Logo.test.tsx) in the front-end.

3. Completely ditch `pnpm jest` altogether and stick with `Turborepo` tests..
Somehow Turborepo can identify the correct `<root>` in its projects.
`pnpm test` or `pnpm test --filter=./src/web/app` will run without crashing

4. Downgrade back to Nextjs v12.1.4

### PR DESCRIPTION
**Pull request by RC-Lee**
## Issue This PR Addresses
Fixes #3643 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue

## Description
As mentioned in the issue, this will allow the `pnpm jest` script to run without crashing. 

A follow-up issue will be written.

## Steps to test the PR
- `pnpm jest`

### CODE REVIEW

**Comment by RC-Lee** [comment]
Follow up issue #3655

File: jest.config.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ module.exports = {
   ...baseConfig,
   projects: [
     // Our front-end
-    '<rootDir>/src/web/app/jest.config.js',
+    // To do: Add front-end tests
```
Comment (humphd) [inline jest.config.js:7]: Can you add the URL to the issue https://github.com/Seneca-CDOT/telescope/issues/3655 here in this comment too?

**Review by humphd — CHANGES_REQUESTED** [review]

File: jest.config.js (hunk 1 above)
Comment (RC-Lee) [reply jest.config.js]: Yup. Added.

**Review by humphd — APPROVED** [review]
Vercel error is #3310 and not related.

**Review by cindyledev — APPROVED** [review]

**Pull request merged**